recordium/config.py

```diff
@@ -53,11 +53,14 @@ def __getattr__(self, key):
 
     def __setattr__(self, key, value):
         if key in self._config_options:
-            if self.data.get(key, self._config_options[key]) != value:
+            if self.data.get(key) != value:
                 self.data[key] = value
                 self._need_save += 1
         else:
-            super(_Config, self).__setattr__(key, value)
+            if key.upper() != key:
```

Comment:
__setattr__ is called every time a new attribute is added to the instance
So, if I make this exception, and check this, I couldn't use self.data = pickle.load(fh), to create a new attribute in the function code.
This is the reason.